kits/ts/simple/lux/game_constants.ts

```diff
@@ -0,0 +1,71 @@
+/**
```

Comment:
we recommend whenever possible, keep all game constants in a json file. This is how we define our constants for the game engine and if possible could you do that too? The JS kit does that you can follow how it does that. I think you may also need to add an option to the tsconfig

Reply:
This was to add type definition automatically. I didnt know typescript has support for json too! Updated now.

Reply:
yep, typescript will auto like treat json as value and type